go.mod

```diff
@@ -40,3 +40,5 @@ require (
 	golang.org/x/sys v0.23.0 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
+
+replace github.com/kovetskiy/gopencils => github.com/petalmd/gopencils v0.0.0-20240827131356-d8904564d2b8
```

Comment:
Pinged @kovetskiy and it got merged, can you remove the replace? :)

Reply:
Done